releasenotes/notes/admissioncontroller-prepend-env-vars-8d88283457c55d9b.yaml

```diff
@@ -0,0 +1,14 @@
+# Each section from every release note are combined when the
```

Comment:
Please move this file to the release notes directory  of the cluster agent (`releasenotes-dca/notes/`)